src/components/auth/connect/OpenConnectedDevice.svelte

```diff
@@ -0,0 +1,24 @@
+<input
+  type="checkbox"
+  id="open-connected-device-modal"
+  checked
+  class="modal-toggle"
+/>
+<div class="modal">
+  <div class="modal-box w-96 relative text-center">
+    <a href="/" class="btn btn-xs btn-circle absolute right-2 top-2">✕</a>
+    <div>
+      <h3 class="mb-5 text-2xl font-serif">Connect an existing account</h3>
+      <div>
+        <p class="text-sm mb-3">
+          To connect with an existing account, you'll need a device that's
```

Comment:
Shoot! Didn't meant to commit this, rather, meant to suggest the copy change. I just wanted to see it in the actual app to decide whether I liked it.

Sorry about that, @bgins!

That being said, what do you think about this copy? 😆

Reply:
I'm gonna go with this copy change. Changing it should be a quick change!